README.md

```diff
@@ -33,11 +33,18 @@ If you are using `dotnet` or `nuget`, you can use the Azure Artifact Credential
 ### Manual installation: Windows
 
 1. Download the latest release of [Microsoft.NuGet.CredentialProvider.zip](https://github.com/Microsoft/artifacts-credprovider/releases)
-1. Unzip the file
-1. Copy the `netcore` directory from the extracted archive to `$env:UserProfile\.nuget\plugins`
+2. Unzip the file
+3. Copy the `netcore` directory from the extracted archive to `$env:UserProfile\.nuget\plugins` to use `dotnet`
+4. Copy the `netfx` directory from the extracted archive to `$env:UserProfile\.nuget\plugins` to use `nuget`
```

Comment:
Per @nkolev92 , we should not recommend installing the netfx credential provider by default, since NuGet will always prefer a user-installed version over a VS-installed version. Can we drop this line?

Reply:
Yep I'll remove it!